src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -569,7 +569,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                                 End If
 
                                 Dim stepOperand As IOperation = If(stepValue.IsConstant, DirectCast(stepValue, IOperation), New Temporary(SyntheticLocalKind.ForLoopStepValue, BoundFor, stepValue))
-                                statements.Add(New CompoundAssignment(controlReference, stepOperand, Semantics.Expression.DeriveAdditionKind(controlType), Nothing, stepValue.Syntax))
+                                statements.Add(New ExpressionStatement(controlReference, stepOperand, Semantics.Expression.DeriveAdditionKind(controlType), Nothing, stepValue.Syntax))
```

Comment:
> I dont know either. that was existing behavior. this I just renamed type name

Why are we merging PR if the answer to the question is: "I dont know either."
Could you please elaborate what did you renamed? And why is it the right thing to do to create an ExpressionStatement instead of an CoumpoundAssignment node in this place?